drafts/inlinemacros.md

```diff
@@ -80,39 +86,37 @@ by-value parameters and that are not side-effect free paths. Arguments to by-nam
 parameters are left alone.
```

Comment:
Therefore, how about we consistently do the introduction of temporary variables all the time. Would our optimizer handle the influx of garbage temporaries?